backend/models/user.py

```
@@ -0,0 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import Optional
+from datetime import datetime
+from bson import ObjectId
```

Ruff

`bson.ObjectId` imported but unused

Remove unused import: `bson.ObjectId`

(F401)